unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/EntityInformation/SmartItems/Parameters/Actions/SmartItemActionEventAdapter.cs

```diff
@@ -11,17 +11,54 @@ public class SmartItemActionEventAdapter : MonoBehaviour
 {
     public TMP_Dropdown entityDropDown;
     public TMP_Dropdown actionDropDown;
+    public TMP_Dropdown optionsDropDown;
     public SmartItemListView smartItemListView;
 
-    SmartItemActionEvent actionEvent;
+    public System.Action<SmartItemActionEventAdapter> OnActionableRemove;
 
-    SmartItemComponent selectedComponent;
-    List<DCLBuilderInWorldEntity> filteredList = new List<DCLBuilderInWorldEntity>();
+
+    private SmartItemActionEvent actionEvent;
+
+    private DCLBuilderInWorldEntity selectedEntity;
+    private List<DCLBuilderInWorldEntity> filteredList = new List<DCLBuilderInWorldEntity>();
 
     private void Start()
     {
         entityDropDown.onValueChanged.AddListener(SelectedEntity);
         actionDropDown.onValueChanged.AddListener(GenerateParametersFromIndex);
+        optionsDropDown.onValueChanged.AddListener(OptionSelected);
+        optionsDropDown.SetValueWithoutNotify(-1);
+    }
+
+    private void OptionSelected(int index)
+    {
+        switch (index)
+        {
+            case 0:
+                ResetActionable();
+                break;
+            case 1:
+                RemoveActionable();
+                break;
+        }
```

Comment:
Not a huge fan of this since a change in the order of the dropdown would require code changes. Buuuut, I havent worked with the `TMP_Dropdown` and maybe there's no alternative.

Reply:
Since this Dropdown is not generated by code the only thing you can do is to add an enum but you will need to change it anyway if you change the order. 

This approach is not "Perfect" since we could create custom actions and it will be better, however, since this won't change (most likely) I think we should be good